src/lib/y2users/linux/login_config_writer.rb

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+
+module Y2Users
+  module Linux
+    # Configures the login settings in the target system according to a given {LoginConfig} object
+    class LoginConfigWriter
+      Yast.import "Autologin"
+
+      # Constructor
+      #
+      # @param login [LoginConfig] see #login_config
```

Comment:
I think below is mentioned also nil as possible value when nothing is to write. So just add it to possible types.

Reply:
I will. But, again, let's try to not re-review code that was just moved from the original `Linux::Writer` to other files. We may loose the focus.